x-pack/plugins/apm/public/tutorial/tutorial_fleet_instructions/index.tsx

```diff
@@ -42,6 +42,7 @@ function TutorialFleetInstructions({ http, basePath, isDarkTheme }: Props) {
         const response = await http.get('/api/apm/fleet/has_data');
         setData(response as APIResponseType);
       } catch (e) {
+        setIsLoading(false);
```

Comment:
Good catch adding this. We should also let the user know that something went wrong but not for this PR